core/templates/components/forms/schema-based-editors/schema-based-list-editor.directive.html

```diff
@@ -49,7 +49,7 @@
             ng-if="!len && (maxListLength === null || localValue.length < maxListLength)"
             type="button" class="btn btn-secondary btn-sm ml-0 protractor-test-add-list-entry" ng-click="addElement()"
             ng-disabled="isDisabled()">
-      <[addElementText]>
+      <[addElementText | translate]>
```

Comment:
Do we support i18n for all the interaction using list-editor?

Reply:
In case a key is not 'translatable', the default behavior of the $translate is to show the key.